src/addons/dragAndDrop/DropWrappers.js

```diff
@@ -151,6 +153,7 @@ function createDropWrapper(type) {
           onEventDrop({
             event,
             ...getEventDropProps(start, end, value, droppedInAllDay),
+            [resourceAccessor]: resource,
```

Comment:
`resourceAccessor` may be function, i would just call this `resource` since it's not hte event object.

Reply:
@jquense Thanks!
Every accessors seem to be String.
https://github.com/intljusticemission/react-big-calendar/pull/833/files#diff-6cf28b0e3b0b32fa18e19efed78ee8b8R80


And... I expect `start` and `end` should support Accessor however, I didn't fix this PR. I think we have to fix on another issue.

Reply:
the defaults are strings but the prop type allows functions. The other ones are already correct, and don't assume it's a string. It's just your usage here that's not right.

Reply:
@jquense Sorry for replying so late. 
I'll check and solve that on this week.

Reply:
@jquense 

I have a question.
Do I have to support resourceAccessor on DnD?

`start` and `end` seem to be fixed.
https://github.com/mura-/react-big-calendar/blob/614f01704c828f0f500ba166d13308daff75650c/src/addons/dragAndDrop/DropWrappers.js#L32

If I don't have to support resourceAccessor, What should I use the resource key name?

Reply:
you only need it if you actually need to get the value from the user's event object. the returned value here is **not** an event object iso you don't need it here. I'd just call this `resourceId` if that's what it is

Reply:
@jquense Thanks a lot. I will fix it.